containers/wordpress/README.md

```diff
@@ -31,8 +31,8 @@ a baseline. The following modifications were made to wp_base in addition to cont
   * NUMA optimization/multi instance (must be done via pinning, see below)
     * Note that for NUMA optimization/pinning you may do this with the base container if you wish to isolate this optimization.
 
-Note that in order to run a baseline across multiple sockets, you will need to utilize the 1s-bkm.js file in the base user 
-directory in the container you wish to run (likely base).  Copy the file over the current my.cnf as shown in the dockerfile.  
+Note that in order to run a baseline across multiple sockets, you will need to utilize the 1s-bkm.js file in the base user
+directory in the container you wish to run (likely base).  Copy the file over the current my.cnf as shown in the dockerfile.
```

Comment:
Can you please clean up newlines, etc so they don't count as changes?

Reply:
The editor I used automatically removed the trailing spaces when saved the file, so it counts as changes.

Reply:
Ah I see, this is probably better then